src/lib/data/data_types.ts

```diff
@@ -56,14 +56,21 @@ interface Budget extends Comments, Document {
 type UserType = 'RSO' | 'SEPC' | 'SGA_Treasurer' | 'SGA_Assistant_Treasurer';
 // User represents a single user
 interface User extends Document {
-  user_name: string;
+  id: string;
+  name: string;
```

Comment:
While I agree that the renaming makes things clearer, this change caused a lot of the tests to fail since those tests assume the user_name field exists. Unless there is good reason to rename our types, please avoid doing so or make sure all tests pass

Reply:
I made the change because the test_user in the Firebase seemed not to have a user_name field but rather a name field.  I discovered this when I tried making the sidebar display the username.  When I tried to display `user.user_name`, it gave me an empty string.  When I tried `user.name`, it worked.  I think it would make more sense to change all the uses of `User.user_name` to `User.name`, but if not, then the Firebase database would have to be changed so that the test user has a `user_name` field rather than a `name` field.